src/commcare_cloud/commands/deploy/commcare.py

```diff
@@ -239,24 +239,23 @@ def call_record_deploy_success(environment, context, end_time):
 def get_deployed_version(environment, from_source=False):
     if from_source:
         release = environment.remote_conf.code_source
+        hosts = "webworkers,celery,proxy,pillowtop,django_manage"
     else:
         release = environment.remote_conf.code_current
+        hosts = "django_manage"
     code_current = shlex.quote(release)
     res = run_ansible_module(
         AnsibleContext(None, environment),
-        "django_manage",
+        hosts,
         "shell",
         f"sudo -iu cchq bash -c 'git --git-dir={code_current}/.git rev-parse HEAD'",
         become=False,
         run_command=ansible_json,
     )
-    result = next(iter(res.values()), {"stderr": "no result for host"})
-    if "stdout" in result:
-        return result["stdout"]
-    error = result["stderr"] if "stderr" in result else repr(result)
-    if "rc" in result:
-        error += f"\n\nreturn code: {result['rc']}"
-    raise BadAnsibleResult(error)
+    versions = {host_result.get('stdout') for host_result in res.values() if host_result.get('stdout')}
```

Comment:
~Looks like this will skip results that do not contain `'stdout'`, which implies it skips hosts that had an error? That seems like the wrong thing to do in the case of an error.~

After reviewing further I think I understand. This facilitates

https://github.com/dimagi/commcare-cloud/blob/ad54a8081488aede6852e28b9992475ff5fdc323/src/commcare_cloud/ansible/roles/deploy_hq/tasks/get_code_version.yml#L29-L30

Would it make sense to use a return code check here to be more like the code this is replacing:
```suggestion
    versions = {host_result.get('stdout') for host_result in res.values() if host_result.get('rc') == 0}
    if not versions or len(versions) > 1:
```
as in

https://github.com/dimagi/commcare-cloud/blob/ad54a8081488aede6852e28b9992475ff5fdc323/src/commcare_cloud/ansible/roles/deploy_hq/tasks/get_code_version.yml#L19

Reply:
559af0d6ae9a83478c3a8fc2ac22b8dfb62b5609